resip/recon/ConversationManager.hxx

```diff
@@ -483,6 +486,18 @@ public:
    virtual void onDtmfEvent(ParticipantHandle partHandle, int dtmf, int duration, bool up) = 0;
 
    ///////////////////////////////////////////////////////////////////////
+   // Pager Message Handlers /////////////////////////////////////////////
+   ///////////////////////////////////////////////////////////////////////
+
+   virtual void onMessageArrived(resip::ServerPagerMessageHandle handle, const resip::SipMessage& message) = 0;
```

Comment:
The out-of-dialog MESSAGE request is already handle when I made ConversationManager derives from ServerPagerMessageHandler and ClientPagerMessageHandler and implement it's methods, right?
Is there an example on how to handle in-dialog MESSAGE requests and how to create a dialog from a out-of-dialog MESSAGE?